src/containers/Home.tsx

```diff
@@ -1,63 +1,94 @@
 import classNames from 'classnames';
-import { Flag } from 'flag';
 import * as React from 'react';
+import MediaQuery from 'react-responsive';
 import { Link } from 'react-router-dom';
-import { getApiCategoryOrder, getApiDefinitions } from '../apiDefs/query';
-import CardLink from '../components/CardLink';
 
-import EmbeddedYoutubeVideo from '../components/EmbeddedYoutubeVideo';
+import { Flag } from 'flag';
+import apiDefinitions, { apiCategoryOrder } from '../apiDefs/data/categories';
+import padlockImg from '../assets/homepage-padlock.png';
+import apiImg from '../assets/homepage-reliable-api.png';
+import CardLink from '../components/CardLink';
+import Hero from '../components/Hero';
+import { OVER_LARGE_SCREEN_QUERY } from '../types/constants';
 import './Home.scss';
 
 class Home extends React.Component {
   public render() {
-    const apiDefinitions = getApiDefinitions();
-    const apiCategoryOrder = getApiCategoryOrder();
+    const leftColumnClasses = classNames('medium-screen:vads-l-col--4', 'small-screen:vads-l-col--6', 'va-api-u-margin-y--auto');
+    const rightColumnClasses = classNames('medium-screen:vads-l-col--8', 'small-screen:vads-l-col--6', 'va-api-u-margin-y--auto', 'va-api-margin-y--auto');
+    const flexContainer = classNames('vads-l-grid-container', 'vads-u-margin-top--6', 'vads-u-margin-x--auto');
```

Comment:
```suggestion
    const flexContainer = classNames('vads-l-grid-container', 'vads-u-margin-top--6');
```

i know that this is how we currently have our home page and the design shows it, but should we be overriding the default grid styles? is the home page special?

Reply:
It makes sense to me to override the default grid for the homepage. The homepage in this PR doesn't look as good left aligned vs centered but it isn't _that_ bad left aligned.

Reply:
ok then, this seems like the right way to do it! no objections, thanks for clarifying